modules/post/hardware/automotive/can_flood.rb

```diff
@@ -0,0 +1,43 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Post
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'can_flood',
+        'Description' => 'Module that floods a CAN interface',
+        'License' => MSF_LICENSE,
+        'Author' => ['Pietro Biondi'],
+        'DisclosureDate' => ['March 20 2019'],
+        'Platform' => ['hardware'],
+        'SessionTypes' => ['hwbridge']
+      )
+    )
+    register_options(
+      [
+        OptInt.new('ROUND_NUMBER', [false, 'Number of executed rounds', 200]),
+        OptString.new('CANBUS', [false, 'CAN interface', nil]),
+        OptString.new('FRAMELIST', [true, 'Path to FRAMELIST', ::File.join(Msf::Config.data_directory, 'wordlists', 'frameListCanBus.txt')])
+      ]
+    )
+  end
+
+  def run
+    print_status(' -- OPENING FRAMELIST FILE --')
```

Comment:
This seems a little unnecessary. If you want to keep it, consider changing `print_status` to `vprint_status`, and optionally also printing the file path.

```suggestion
    vprint_status("Reading frame list file: #{datastore['FRAMELIST']}")
```

Reply:
Thanks for the tips